tests/rt.sh

```diff
@@ -99,12 +99,7 @@ source rt_utils.sh
 
 if [[ $MACHINE_ID = wcoss_cray ]]; then
 
-  source $PATHTR/NEMS/src/conf/module-setup.sh.inc
```

Comment:
My understanding is that this line actually does "module purge", which cleans up users' environment modules loaded before they run rt.sh. May I ask the reason we remove them?

Reply:
I'm not sure about this line, maybe we can leave it, but fv3 module should not be loaded in the main script because ecflow server process remembers all the modules loaded before it's started.

Reply:
Thanks, I think it is fine to remove loading the fv3 module  as we have separate module load at compile time and at run time.